src/server/game/Instances/InstanceScript.cpp

```diff
@@ -153,6 +153,53 @@ void InstanceScript::LoadBossBoundaries(BossBoundaryData const& data)
             bosses[entry.BossId].boundary.push_back(entry.Boundary);
 }
 
+void InstanceScript::LoadMinionData(MinionData const* data)
```

Comment:
The current existing InstanceScript::LoadMinionData just replaces this one from TC in a nicer way while staying compatible.